### PR TITLE
use central package management

### PR DESCRIPTION
**Pull request by jmarolf**

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  
+  <PropertyGroup>
+    <MicrosoftBuildVersion>15.3.409</MicrosoftBuildVersion>
+    <MicrosoftExtensionsVersion>5.0.0-preview.7.20364.11</MicrosoftExtensionsVersion>
+    <!-- In order tests against the same version of NuGet as the SDK. We have to set this to match. -->
+    <NuGetVersion>5.8.0-preview.3.6823</NuGetVersion>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageVersion Include="BenchmarkDotNet.Annotations" Version="0.12.1" />
```
Comment (JoeRobich) [inline Directory.Packages.props:12]: Should these be updates not includes?

Comment (jmarolf) [reply Directory.Packages.props:12]: nope the default targets already take `PackageVersion` metadata and update `PackageReference` versions.

File: .gitignore [hunk 2]
```diff
@@ -24,6 +24,7 @@ bld/
 [Oo]bj/
 [Ll]og/
 [Tt]emp/
+**/packages.lock.json
```
Comment (RikkiGibson) [inline .gitignore:27]: did you try using the central lockfile? or does it just feel like lockfiles still change too often to be pleasant to use here?

**Review by RikkiGibson — APPROVED** [review]

**Review by JoeRobich — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (jmarolf) [reply .gitignore:27]: lock files are always generated for central package management

**Pull request merged**